pkg/apis/serving/v1alpha1/llm_inference_service_lifecycle.go

```diff
@@ -126,7 +134,3 @@ func (in *LLMInferenceService) MarkSchedulerWorkloadNotReady(reason, messageForm
 func (p *InferencePoolSpec) HasRef() bool {
 	return p != nil && p.Ref != nil && p.Ref.Name != ""
 }
-
```

Comment:
I think we can also move `func (p *InferencePoolSpec) HasRef() bool` to `_types.go`

Reply:
The idea is to make it easier to upstream changes as this file is net new

Reply:
Good point. then perhaps we can move it to llm_inference_service_ext_func.go as these are not related to lifecycle has such?